test/extended/builds/build_pruning.go

```diff
@@ -149,4 +152,34 @@ var _ = g.Describe("[builds][pruning] prune builds based on settings in the buil
 
 	})
 
+	g.It("buildconfigs should have a default history limit set when created via the group api", func() {
```

Comment:
they're "relatively" cheap.  they all involve running builds which is never less than a minute or so, even though we're not pushing anywhere.

anyway i did end up adding a unit test, so at least if someone breaks the strategy we'll know.  if they stop *using* the strategy, we'll find out when the nightly run fails.